content/en/docs/concepts/extend-kubernetes/poseidon-firmament-alternate-scheduler.md

```diff
@@ -6,9 +6,11 @@ weight: 80
 
 {{% capture overview %}}
 
-{{< feature-state for_k8s_version="v1.13" state="alpha" >}}
+{{< note >}}
+	 Current release of Poseidon-Firmament scheduler is an alpha release.
+{{< /note >}}
```

Comment:
@anvithks, here are some more `suggestions`. I am just reviewing; doc maintainers approve.
If the note content is moved to the left, I think the rendered output may look better. You may need to test this.

Reply:
The note is a custom Hugo shortcode. I am not sure if I can change the style. I will check.
Maybe I can just display the text emboldened instead of using note.

Reply:
Changed it to emboldened text.